compiler/rustc_middle/src/ty/sty.rs

```diff
@@ -955,7 +955,7 @@ impl<'tcx> PolyExistentialTraitRef<'tcx> {
 /// erase, or otherwise "discharge" these bound vars, we change the
 /// type from `Binder<T>` to just `T` (see
 /// e.g., `liberate_late_bound_regions`).
```

Comment:
```suggestion
/// e.g., `liberate_late_bound_regions`).
///
/// `Decode` and `Encode` is implemented for `Binder<T>` using the `impl_binder_encode_decode!` macro. 
```
or something like that